src/main/java/com/oltpbenchmark/ThreadBench.java

```diff
@@ -101,6 +104,38 @@ private int finalizeWorkers(ArrayList<Thread> workerThreads) throws InterruptedE
         return requests;
     }
 
+    private Results runReplayBenchmark() {
+        // IGNORE THIS: boilerplate to set up a phase
+        WorkloadConfiguration workConf = this.workConfs.get(0);
+        workConf.addPhase(0, 0, 0, 0, new ArrayList<Double>(), true, false, false, false, 0, Arrival.REGULAR);
+        workConf.initializeState(testState);
+        WorkloadState workState = workConf.getWorkloadState();
+        this.createWorkerThreads();
+        workState.switchToNextPhase(); // switch to the phase we just added
+        // END IGNORE THIS
+
+
+        // CORE CODE: demos passing a sql statement
+        List<SubmittedProcedureRun> submittedProcedureRuns = new ArrayList<SubmittedProcedureRun>();
+        List<Object> sqlStmts = new ArrayList<Object>();
+        sqlStmts.add(new SQLStmt("SELECT * FROM customer LIMIT 10;"));
```

Comment:
This looks reasonable to me. You may need to think about prepared statements for OLTP, or maybe not -- maybe https://jdbc.postgresql.org/documentation/server-prepare/#server-prepared-statements is enough.

Reply:
Thanks for the link. I think if prepared statements can't be folded into the list of sqlStmts we can just pass in a list of prepared statements as the first "Object" and a list of sqlStmts as the second "Object"